### PR TITLE
Staging redirectoraddr

### PR DESCRIPTION
**Pull request by yashvardhan-netex**
- Added helper script to apply redirector address
 - Added support to preserve the same

### CODE REVIEW

File: feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c [hunk 1]
```diff
@@ -477,3 +477,21 @@ int ieee80211_channel_to_frequency(int chan)
...
 	return 0;
 }
+
+int get_redirector_addr(char *addr)
+{
+	char filepath[PATH_MAX] = {0};
+	char buf[REDIRECTOR_ADDR_SIZE];
+	int fd, ret;
+
+	snprintf(filepath, sizeof(filepath), "/usr/opensync/etc/redirector");
```
Comment (blogic) [inline feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c:487]: I would prefer this to be stored in uci, I will prepare an according patch for testing
   John

Comment (yashvardhan-netex) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c:487]: Hi John,
It totally makes sense to use a UCI parameter. If you have not already done the changes, should I take it up?

Comment (blogic) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c:487]: I have not started yet, so go ahead please

Comment (yashvardhan-netex) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c:487]: Sure. I will do that. Thanks!

**Comment by blogic** [comment]
merged into pending, Thanks !

**Pull request closed without merging**